src/AWS/Signers/V4.elm

```diff
@@ -52,7 +54,7 @@ formatDate date =
     date
         |> formatISO8601
         |> Regex.replace All
-            (regex "([-:\\]|\\.\\d{3})")
+            (regex "[-:]")
```

Comment:
I admittedly simplified this and it may not be ideal. I was getting runtime errors in the tests from the original.